gensim/summarization/textcleaner.py

```diff
@@ -22,28 +29,102 @@
 
 
 SEPARATOR = r'@'
-RE_SENTENCE = re.compile(r'(\S.+?[.!?])(?=\s+|$)|(\S.+?)(?=[\n]|$)', re.UNICODE)
+"""str: special separator used in abbreviations."""
+RE_SENTENCE = re.compile(r'(\S.+?[.!?])(?=\s+|$)|(\S.+?)(?=[\n]|$)', re.UNICODE)  # backup (\S.+?[.!?])(?=\s+|$)|(\S.+?)(?=[\n]|$)
+"""SRE_Pattern: pattern to split text to sentences."""
```

Comment:
Problem with building here

Reply:
@yurkai good example, how to document it: https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/parsing/preprocessing.py#L21